docs/governance/templates/release_issue.md

```diff
@@ -0,0 +1,54 @@
+# Release {version}
+
+<!--
+This is the release issue template. Make a copy of the markdown in this page
+and copy it into a release issue. Fill in relevent values, found inside {}
+!-->
+Complete Milestone
+------------------
+
+- [ ] Create the next version milestone, use [semantic versioning](https://semver.org/) when naming it to be consistent with the community.
+- [ ] Visit the [milestone](https://github.com/GoogleCloudPlatform/open-match/milestone).
+  - [ ] Review closed issues have appropriate tags.
+  - [ ] Review closed issues have been applied to the current milestone.
+  - [ ] Review closed PRs have appropriate tags.
```

Comment:
What are the appropriate tags?  If we don't have a guide yet, you can include a file with just a todo on me and I'll fill it out.